Week1/homework/app.js

```diff
@@ -2,10 +2,104 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'dont_tell_me_your_afraid', 'dont_tell_mummy', 'positive_thinking', 'time_management', 'The alchemist', 'the_power_of_now', 'ekadoli', 'Before_we_were_yours', 'hebrew_female_in_my_heart', 'amareeta'
   ];
 
-
   // Replace with your own code
-  console.log(bookTitles);
+  //  2.3
+  function makeBooksTitlesList() {
+    const ul = document.createElement('ul');
+    ul.innerHTML = 'Books Titles: ';
+    document.getElementsByTagName('body')[0].appendChild(ul);
+    for (let i = 0; i < bookTitles.length; ++i) {
+      const li = document.createElement('li');
+      li.innerHTML = bookTitles[i];
+      ul.appendChild(li);
+    }
+
+  }
+
+  makeBooksTitlesList();
```

Comment:
It is is best to place this function call inside of a function `main` along with the function call in 103. Then you could assign function `main` to `window.onload` like this:

```js
window.onload = main;
```

This ensure that you code is executed when the web page is fully loaded.